src/core/public/chrome/ui/header/branding/opensearch_dashboards_custom_logo.tsx

```diff
@@ -32,13 +32,16 @@
 
 import React from 'react';
 import '../header_logo.scss';
+import { OpenSearchDashboardsLogoDarkMode } from './opensearch_dashboards_logo_darkmode';
 
 export interface CustomLogoType {
-  logoUrl: string;
+  logoUrl: string | undefined;
 }
 
 export const CustomLogo = ({ ...branding }: CustomLogoType) => {
-  return (
+  return branding.logoUrl === undefined ? (
```

Comment:
i wonder if we can use the logical equivalent for no-value (empty string, null, undefined)

return !branding.logoUrl ? OpenSearchDashboardsLogoDarkMode() : <img ...>

Reply:
Yes, will change to logical equivalent so it looks cleaner.

Reply:
Changed all "... === undefined" to logical equivalent for consistency.